pkg/features/kube_features.go

```diff
@@ -652,6 +652,7 @@ const (
 
 	// owner: @ehashman
 	// alpha: v1.21
+	// beta: v1.22
```

Comment:
```suggestion
	// beta: v1.25
```

Reply:
It actually graduated to beta in 1.22. We just set the default value to true in 1.25.

Reply:
got it... so maybe `// beta: v1.22 (off by default until 1.25)` ? as per convention in https://github.com/kubernetes/kubernetes/blame/master/pkg/features/kube_features.go#L258

Reply:
We covered this description below in L990.